iceoryx_hoofs/include/iceoryx_hoofs/cxx/helplets.hpp

```diff
@@ -338,7 +338,14 @@ bool doesEndWithPathSeparator(const string<StringCapacity>& name) noexcept;
 /// @param[in] value of type F to convert to T
 /// @return converted value of F to corresponding value of T
 template <typename F, typename T>
-constexpr T from(const F value) noexcept;
+constexpr T from(const F value);
```

Comment:
It can't be guaranteed that types `T` and `F` don't throw, hence I removed `noexcept` everywhere.